### PR TITLE
Specify minimum Vagrant version to 2.2.10 Issue #159

### PR DESCRIPTION
**Pull request by rliberma**

### CODE REVIEW

**Review by IlanZuckerman — APPROVED** [review]
lgtm

**Review by lioramilbaum — CHANGES_REQUESTED** [review]
Please link the issue

File: Vagrantfile [hunk 1]
```diff
@@ -9,6 +9,9 @@ class Hash
   end unless Hash.method_defined?(:except)
 end
 
+# Specify minimum Vagrant version
+Vagrant.require_version '>= 2.2.10'
```
Comment (lioramilbaum) [inline Vagrantfile:13]: This check will pass when 2.2.11 is released. Not sure we want it.

**Review by lioramilbaum — CHANGES_REQUESTED** [review]

File: Vagrantfile (hunk 1 above)
Comment (rliberma) [reply Vagrantfile]: @lioramilbaum Updated and lock to 2.2.10 now. All checks have passed. Please approve. Thanks.

**Review by IlanZuckerman — APPROVED** [review]
LGTM

**Review by lioramilbaum — APPROVED** [review]

**Pull request merged**